packages/commonwealth/server/routes/bulkTopics.ts

```diff
@@ -1,16 +1,27 @@
 import type { Request, Response } from 'express';
 import type { DB } from '../models';
+import { QueryTypes } from 'sequelize';
 
 export const Errors = {};
 
 const bulkTopics = async (models: DB, req: Request, res: Response) => {
   const chain = req.chain;
 
-  const topics = await models.Topic.findAll({
-    where: { chain_id: chain.id },
-  });
+  const topics = await models.sequelize.query(
+    `SELECT
+        *,
+        (
+          SELECT COUNT(*)::int FROM "Threads"
+          WHERE chain = :chain_id AND topic_id = t.id AND deleted_at IS NULL
+        ) as total_threads
+      FROM "Topics" t WHERE chain_id = :chain_id AND deleted_at IS NULL`,
+    {
+      replacements: { chain_id: chain.id },
+      type: QueryTypes.SELECT,
+    }
```

Comment:
moved this from `bulkOffchain` to make sure that https://github.com/hicommonwealth/commonwealth/pull/3894 still works correctly